projects/storefrontlib/src/cms-components/checkout/cart/add-to-cart/added-to-cart-dialog/added-to-cart-dialog.component.ts

```diff
@@ -62,9 +62,9 @@ export class AddedToCartDialogComponent implements OnInit, AfterViewChecked {
 
           // Announce in header if Add To Cart button has incremented product
           if (this.firstUpdate && entry.quantity > 1) {
-            this.headerLabel = `addToCart.itemsIncrementedInYourCart`;
+            this.showItemIncrLabel = true;
```

Comment:
it doesn't look good.
```ts
this.showItemIncrLabel = this.firstUpdate && entry.quantity > 1
```
Why you didn't use sample from description?

Reply:
It does not work because the firstUpdate flag is already invalidated by the time entry$ observable is returned.

Reply:
Confirmed. So lets remove this "if" condition for now only.